tests/testsuite.src/syn_file.at

```diff
@@ -680,14 +688,15 @@ AT_DATA([prog.cob], [
 ])
 
 # FIXME: "is not defined" should be changed in "is not defined in file ..."
-AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
+AT_CHECK([$COMPILE_ONLY -Wno-unfinished prog.cob], [1], [],
```

Comment:
I suggest using a numeric binary instead of the bit item - that should raise the same message and is fully implemented, no?

Reply:
Actually this is for `prog.cob:30: warning: handling of FILE COLLATING SEQUENCE is unfinished; implementation is likely to be changed`. Not sure what's best here: silence the warning? or include it in the test output?

Reply:
Make a conditional to expect for BDB, not in other cases (like the 32bit test, depending on "failure" or "pass" run the actual test)?